demo/api/src/products/coordinates.type.ts

```diff
@@ -11,12 +11,20 @@ export class Coordinates {
     longitude: number;
 }
 
-export class CoordinatesType extends Type<Coordinates, string> {
-    convertToDatabaseValue(value: Coordinates): string {
+export class CoordinatesType extends Type<Coordinates | null, string | null> {
```

Comment:
MikroORM has a similar example in their docs, where they use undefined instead of null:

https://mikro-orm.io/docs/custom-types#advanced-example---pointtype-and-wkt

However, the value passed to convertToDatabaseValue is null, and their example only works because of using a falsy check `if(!value)`. Should I still change this to undefined?

Reply:
if it is null at runtime, using null is correct (we don't do that for entities usually though)

I would use a falsy check, that doesn't do any harm

Reply:
Changed: e93c962e833312e919b32835b91828f3fa7ea4d7